app/serializers/framework_question_serializer.rb

```diff
@@ -1,9 +1,24 @@
 class FrameworkQuestionSerializer < ActiveModel::Serializer
   belongs_to :framework
 
-  attributes :key, :section, :question_type, :options
+  attributes :key, :section, :question_type, :options, :response_type
+  has_many :dependents, key: :descendants
+
+  def response_type
+    case object.question_type
+    when 'radio'
+      object.followup_comment ? 'object' : 'string'
+    when 'checkbox'
+      object.followup_comment ? 'collection' : 'array'
+    when 'add_multiple_items'
+      'collection::add_multiple_items'
```

Comment:
We'll be changing other types to be more explicit, but that requires a timed deploy, so only amending the new type till then